pkg/controller/contrailmonitor/contrailmonitor_controller.go

```diff
@@ -0,0 +1,453 @@
+		return controllerutil.SetControllerReference(instance, serIns6, r.scheme)
+	})
+	if err != nil {
+		return reconcile.Result{}, nil
+	}
+
+	zookeeper, err := r.getZookeeper(instance)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	zookeeperActive := zookeeper.IsActive(instance.Spec.ServiceConfiguration.ZookeeperInstance, request.Namespace, r.client)
+	serIns5 := &contrailv1alpha1.Contrailstatusmonitor{ObjectMeta: metav1.ObjectMeta{Name: zookeeper.Name, Namespace: "contrail"}}
+
+	if zookeeper.Status.Active == nil || !zookeeperActive {
+		serIns5.Status = "NotActive"
+	} else {
+		serIns5.Status = "Active"
+	}
+	_, err = controllerutil.CreateOrUpdate(context.Background(), r.client, serIns5, func() error {
+		return controllerutil.SetControllerReference(instance, serIns5, r.scheme)
+	})
+	if err != nil {
+		return reconcile.Result{}, nil
+	}
+
+	cassandra, err := r.getCassandra(instance)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	cassandraActive := cassandra.IsActive(instance.Spec.ServiceConfiguration.CassandraInstance, request.Namespace, r.client)
+	serIns4 := &contrailv1alpha1.Contrailstatusmonitor{ObjectMeta: metav1.ObjectMeta{Name: cassandra.Name, Namespace: "contrail"}}
+	if cassandra.Status.Active == nil || !cassandraActive {
+		serIns4.Status = "NotActive"
+	} else {
+		serIns4.Status = "Active"
+	}
+	_, err = controllerutil.CreateOrUpdate(context.Background(), r.client, serIns4, func() error {
+		return controllerutil.SetControllerReference(instance, serIns4, r.scheme)
+	})
+	if err != nil {
+		return reconcile.Result{}, nil
+	}
+
+	wklist, err := r.getWebuilist()
+	fmt.Println(wklist.Items[0].Status.ServiceStatus)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	wcount := len(wklist.Items)
+	if wcount > 0 {
+		for i := 0; i < wcount; i++ {
+			for _, value := range wklist.Items[i].Status.ServiceStatus {
+				for _, n := range value {
+					serIns8 := &contrailv1alpha1.Contrailstatusmonitor{ObjectMeta: metav1.ObjectMeta{Name: n.ModuleName, Namespace: "contrail"}}
+
+					serIns8.Status = n.ModuleState
+					_, err := controllerutil.CreateOrUpdate(context.Background(), r.client, serIns8, func() error {
+						return controllerutil.SetControllerReference(instance, serIns8, r.scheme)
+					})
+					if err != nil {
+						return reconcile.Result{}, err
+					}
+				}
+			}
+		}
+	} else {
+
+		serIns8 := &contrailv1alpha1.Contrailstatusmonitor{ObjectMeta: metav1.ObjectMeta{Name: "Webui", Namespace: "contrail"}}
+
+		serIns8.Status = "NotActive"
+		_, err := controllerutil.CreateOrUpdate(context.Background(), r.client, serIns8, func() error {
+			return controllerutil.SetControllerReference(instance, serIns8, r.scheme)
+		})
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+
+	}
+
+	clist, err := r.getConfiglist()
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	fmt.Println(clist.Items[0].Status.ServiceStatus)
```

Comment:
It looks like a debug log. Please use logger or remove it

Reply:
I will remove it.